### PR TITLE
feat: adding Back to top button

### LINKED ISSUE #281: [FEATURE] Adding back-to-top button
### Description

As the content increases on site, it hard and boring step to scroll back-to-top manually with scrollbar, and espacially in projects page  as number of projects will grows it would be nice to have a back-to-top button.

### PR DESCRIPTION
**Pull request by rupali-codes**
### Fixes Issue

Closes #281 

### Changes proposed
- added BackToTopButton component
- importing this component into Footer for home and other pages, aslo into projects page 

### Screenshots
image

image

### CODE REVIEW

**Comment by rohitdasu** [comment]
@rupali-codes i've 3 change requests...
1. please fix the conflicts in the branch
2. when i scroll down to bottom of the page, i'm unable to click the backToTop button.
3. can we restrict the backToTop button at homepage and projects only?

**Comment by rupali-codes** [comment]
sure @rohitdasu, making my changes

File: components/BackToTopButton/BackToTop.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { FaArrowUp } from 'react-icons/fa';
+
+export const BackToTop = () => {
+  const handleClick = () => {
+    console.log('clicked');
```
Comment (rohitdasu) [inline components/BackToTopButton/BackToTop.tsx:6]: please remove console message

Comment (rupali-codes) [reply components/BackToTopButton/BackToTop.tsx:6]: ok

**Review by rohitdasu — APPROVED** [review]

**Pull request merged**